CONTRIBUTORS.md

```diff
@@ -0,0 +1,9 @@
+Aion foundation
+<ether.camp> team through the ethereumJ library.
+Ether.Camp Inc. (US) team through Ethereum Harmony.
+John Tromp through the Equihash solver.
+Samuel Neves through the BLAKE2 implementation.
+Zcash project team.
+Bitcoinj team.
```

Comment:
We had a few more names on this page: https://github.com/aionnetwork/aion/wiki/Contributors
Also, the wiki page can be removed after this change.